tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -111,6 +111,7 @@ void sourcekitd::initializeService(
     StringRef diagnosticDocumentationPath,
     std::function<void(sourcekitd_response_t)> postNotification) {
   INITIALIZE_LLVM();
+  initializeSwiftParseModules();
   initializeSwiftModules();
```

Comment:
Would it be worth renaming `initializeSwiftModules` to something like `initializeSwiftSILModules` on the Swift side, and then defining `initializeSwiftModules` on the C side to call both that and `initializeSwiftParseModules`?